pkg/features/kube_features.go

```diff
@@ -416,8 +420,8 @@ var defaultKubernetesFeatureGates = map[utilfeature.Feature]utilfeature.FeatureS
 	MountContainers:                             {Default: false, PreRelease: utilfeature.Alpha},
 	VolumeScheduling:                            {Default: true, PreRelease: utilfeature.GA},
 	CSIPersistentVolume:                         {Default: true, PreRelease: utilfeature.Beta},
-	CSIDriverRegistry:                           {Default: false, PreRelease: utilfeature.Alpha},
-	CSINodeInfo:                                 {Default: false, PreRelease: utilfeature.Alpha},
+	CSIDriverRegistry:                           {Default: true, PreRelease: utilfeature.Beta},
+	CSINodeInfo:                                 {Default: true, PreRelease: utilfeature.Beta},
```

Comment:
1st commit moves objects to beta. This commit enables the features that make use of the objects to Beta and enables them by default...
masters upgrade -> CRDs are installed/reconciled by addonmanager -> nodes upgrade

Reply:
nvm want to test but, tests are failing for other reasons -_-